File: jdroid-android-core/src/debug/java/com/jdroid/android/debug/info/DebugInfoActivity.java
package com.jdroid.android.debug.info;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class DebugInfoActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return DebugInfoFragment.class;
	}
}

File: jdroid-android-core/src/debug/java/com/jdroid/android/debug/info/DebugInfoActivity.kt
package com.jdroid.android.debug.info

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class DebugInfoActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment>? {
        return DebugInfoFragment::class.java
    }
}
